Helper/QueryFilterHelper.php

```diff
@@ -220,6 +224,19 @@ private function getCustomValueValueExpression(
                 );
 
                 break;
+            case 'between':
+            case 'notBetween':
+                if (is_array($filterParameterValue)) {
+                    $expression = $customQuery->expr()->{$operator}(
+                        $tableAlias.'_value.value',
+                        array_map(function (mixed $val) use ($customQuery): mixed {
+                            return is_numeric($val) && intval($val) === $val ?
+                                $val : $customQuery->expr()->literal($val);
+                        }, array_values($filterParameterValue))
+                    );
+                    break;
+                }
+                // no break
```

Comment:
We don't need this comment anymore, do we?
```suggestion

```

Reply:
it is failing on fixcs and when running fixcs then it is adding this line again.